lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -4636,6 +4889,7 @@ class DecomposeComplexOpsPass
     addPatternIfTargetOpIsIllegal<DecomposeAtenRandLikeOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenHardsigmoidOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenRelu6Op>(patterns);
+    addPatternIfTargetOpIsIllegal<DecomposeAtenEinsumOp>(patterns);
```

Comment:
Needs to also be added here: https://github.com/llvm/torch-mlir/blob/0caaf8d32abc73e6dee6ff3b39573cc7fdc2929f/lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp#L364